app/gist/new/route.js

```diff
@@ -7,7 +7,7 @@ export default GistRoute.extend({
   model () {
     this.store.unloadAll('gistFile');
 
-    var model = this.store.createRecord('gist', {description: 'New Twiddle'});
+    var model = this.store.createRecord('gist', {description: 'New Twiddle '});
```

Comment:
I tried that before and just tried it again and the title was not cut off. I tried it both logged in and logged out. This issue is unpredictable enough it's almost like a practical joke!